frontend/src/lib/api.ts

```diff
@@ -236,10 +236,15 @@ const api = {
     },
 
     async get(url: string, signal?: AbortSignal): Promise<any> {
+        console.log(url, 'starting')
         if (url.indexOf('http') !== 0) {
-            url = '/' + url + (url.indexOf('?') === -1 && url[url.length - 1] !== '/' ? '/' : '')
-        }
+            if (!url.startsWith('/')) {
```

Comment:
Perhaps a bit of scope creep, but I guess this is also an issue with the other verbs.

Reply:
Although non-blocking, if this solves the immediate issue and updating elsewhere is a more risky task.

Reply:
Not scope creep, good thinking!